modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/jdbc/JdbcQueryCursor.java

```diff
@@ -87,4 +87,18 @@ public SqlQueryType queryType() {
     public ResultSetMetadata metadata() {
         return cur.metadata();
     }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean hasNextResult() {
+        // TODO https://issues.apache.org/jira/browse/IGNITE-20661
+        return false;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public CompletableFuture<AsyncSqlCursor<T>> nextResult() {
+        // TODO https://issues.apache.org/jira/browse/IGNITE-20661
+        throw new UnsupportedOperationException();
```

Comment:
Seems, we can throw correct error here
```suggestion
       if (!hasNextResult()) {
           throw new QueryHasNoMoreResultsException();
       }
```